csci3230-project/server.js

```diff
@@ -1,14 +1,15 @@
 var express = require('express');
```

Comment:
Please add `'use strict';` to the top of the file, you can look up what it does on google, its quite good :smiley: